metricbeat/module/elasticsearch/index/index.go

```diff
@@ -39,7 +39,7 @@ func init() {
 
 const (
 	statsMetrics = "docs,fielddata,indexing,merge,search,segments,store,refresh,query_cache,request_cache"
-	statsPath    = "/_stats/" + statsMetrics + "?filter_path=indices"
+	statsPath    = "/_stats/" + statsMetrics + "?filter_path=indices&expand_wildcards=all"
```

Comment:
`expand_wildcards` defaults to `open`, so this change will add getting stats for `hidden` indices, but also `closed` indices, are we ok with that?

Reply:
Ha, great catch! Let me play with it, especially with the Stack Monitoring UI in the picture, and see what impact it has. Thanks!

Reply:
If we only want to add `hidden` indices I guess we can use `expand_wildcards=open,hidden` and then coordinate with UI to see how to implement the visualization for `closed` indices and add them in a future change.

Reply:
I checked the behavior of the `elasticsearch` module prior to this PR. It does not collect metrics for closed indices. So I'm +1 to maintaining the same behavior with this PR as well. Will update the PR shortly. Thanks, again, @jsoriano for bringing this up!